scenario/cold_migrate.yaml

```diff
@@ -106,3 +106,4 @@ process:
       - rename_info_iter: True
   - verify:
       - verify_vms: True
+      - remove_failed_instances: True
```

Comment:
We have a lot of different scenarios in `CloudFerry/scenario` directory, so it would be nice to add this action to all of those, who migrate VMs.